server/services/ApplicationsDeveloper.js

```diff
@@ -170,15 +170,38 @@ class ApplicationsDeveloper {
   }
 
   async updateKey (data, appId, keyId, keyType, oauthToken, environment) {
-    if (keyType === 'key') {
-      return this.updateRestApiKey(data, appId, keyId, oauthToken, environment);
-    } else if (keyType === 'stream-key') {
+    if (keyType === 'key'){
+      return this.updateRestApiKey(data, appId, keyId, oauthToken, environment)
+    }else if(keyType === 'web'){
+      return this.updateWebKey(data, appId, keyId, oauthToken, environment)
+    }else if (keyType === 'stream-key') {
```

Comment:
Is this spacing around the brackets standard?

Reply:
changed